yew/src/sgml_tags.rs

```diff
@@ -0,0 +1,442 @@
+    "nextid",
+    "nobr",
+    "noembed",
+    "noframes",
+    "plaintext",
+    "shadow",
+    "spacer",
+    "strike",
+    "tt",
+    "xmp",
+];
+
+// Source: https://developer.mozilla.org/en-US/docs/Web/MathML/Element
+static MATHML_TAGS: [&str; 44] = [
+    "annotation-xml",
+    "annotation",
+    "maction",
+    "maligngroup",
+    "malignmark",
+    "math",
+    "menclose",
+    "merror",
+    "mfenced",
+    "mfrac",
+    "mglyph",
+    "mi",
+    "mlabeledtr",
+    "mlongdiv",
+    "mmultiscripts",
+    "mn",
+    "mo",
+    "mover",
+    "mpadded",
+    "mphantom",
+    "mprescripts",
+    "mroot",
+    "mrow",
+    "ms",
+    "mscarries",
+    "mscarry",
+    "msgroup",
+    "msline",
+    "mspace",
+    "msqrt",
+    "msrow",
+    "mstack",
+    "mstyle",
+    "msub",
+    "msubsup",
+    "msup",
+    "mtable",
+    "mtd",
+    "mtext",
+    "mtr",
+    "munder",
+    "munderover",
+    "none",
+    "semantics",
+];
+
+// Source: https://developer.mozilla.org/en-US/docs/Web/SVG/Element
+static SVG_TAGS: [&str; 90] = [
+    "a",
+    "altGlyph",
+    "altGlyphDef",
+    "altGlyphItem",
+    "animate",
+    "animateColor",
+    "animateMotion",
+    "animateTransform",
+    "circle",
+    "clipPath",
+    "color-profile",
+    "cursor",
+    "defs",
+    "desc",
+    "discard",
+    "ellipse",
+    "feBlend",
+    "feColorMatrix",
+    "feComponentTransfer",
+    "feComposite",
+    "feConvolveMatrix",
+    "feDiffuseLighting",
+    "feDisplacementMap",
+    "feDistantLight",
+    "feDropShadow",
+    "feFlood",
+    "feFuncA",
+    "feFuncB",
+    "feFuncG",
+    "feFuncR",
+    "feGaussianBlur",
+    "feImage",
+    "feMerge",
+    "feMergeNode",
+    "feMorphology",
+    "feOffset",
+    "fePointLight",
+    "feSpecularLighting",
+    "feSpotLight",
+    "feTile",
+    "feTurbulence",
+    "filter",
+    "font-face-format",
+    "font-face-name",
+    "font-face-src",
+    "font-face-uri",
+    "font-face",
+    "font",
+    "foreignObject",
+    "g",
+    "glyph",
+    "glyphRef",
+    "hatch",
+    "hatchpath",
+    "hkern",
+    "image",
+    "line",
+    "linearGradient",
+    "marker",
+    "mask",
+    "mesh",
+    "meshgradient",
+    "meshpatch",
+    "meshrow",
+    "metadata",
+    "missing-glyph",
+    "mpath",
+    "path",
+    "pattern",
+    "polygon",
+    "polyline",
+    "radialGradient",
+    "rect",
+    "script",
+    "set",
+    "solidcolor",
+    "stop",
+    "style",
+    "svg",
+    "switch",
+    "symbol",
+    "text",
+    "textPath",
+    "title",
+    "tref",
+    "tspan",
+    "unknown",
+    "use",
+    "view",
+    "vkern",
+];
+
+static DISALLOWED_CUSTOM_ELEMENT_TAGS: Lazy<Mutex<Vec<&str>>> = Lazy::new(|| {
+    Mutex::new(
+        SVG_TAGS
+            .iter()
+            .chain(MATHML_TAGS.iter())
+            .filter(|tag| tag.contains('-'))
+            .map(|t| t.to_owned())
+            .collect(),
+    )
+});
+
+/// Returns true iff the character provided is a "control" as defined
```

Comment:
Although `iff` isn't *wrong*, I think it's clearer to write either "if" or "only if"
```suggestion
/// Returns true only if the character provided is a "control" as defined
```